modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1672,32 +1680,55 @@ public Expression textTranslateAnchor() {
         String symbolPlacementVal =
                 transformer.requireLiteral(
                         symbolPlacement(), String.class, "point", "symbol-placement", getId());
-
+        Expression fontSize = textSize();
         if ("point".equalsIgnoreCase(symbolPlacementVal.trim())) {
             // Point Placement (default)
             PointPlacement pointP = sb.createPointPlacement();
             // Set anchor point (translated by text-translate)
             // GeoTools AnchorPoint doesn't seem to have an effect on PointPlacement
             pointP.setAnchorPoint(anchorPoint());
 
-            // MapBox text-offset: +y means down
-            Displacement textTranslate = textTranslateDisplacement();
-            textTranslate.setDisplacementY(
-                    ff.multiply(ff.literal(-1), textTranslate.getDisplacementY()));
-            pointP.setDisplacement(textTranslate);
-
+            // MapBox text-displacement: +y means down, expressed in px
```

Comment:
This should say "text-translate" to match the property name

Reply:
Right, will fix